compiler-cli/src/fs.rs

```diff
@@ -182,7 +182,7 @@ impl CommandExecutor for ProjectIO {
             .args(args)
             .stdin(stdio.get_process_stdio())
             .stdout(stdio.get_process_stdio())
-            .envs(env.iter().map(|(a, b)| (a, b)))
+            .envs(env.iter().map(|(a, b): &(&str, String)| (a, b)))
```

Comment:
What's this for?

Reply:
I noticed CI was failing because the linter wanted me to remove this map, it's a false positive because it thinks it's the same as applying the identity function so I had to make the type explicit

Reply:
You can alternatively use `#[allow(clippy::map_identity)]`

Reply:
Ah I see, thank you! I'll revert this then